countries.js

```diff
@@ -0,0 +1,87 @@
+document.addEventListener("DOMContentLoaded", function() {
+    // Fetch country data asynchronously
+    fetchCountryData()
```

Comment:
I wont repaet myself but the same comments apply in here as well 

1) Remove comment
2) extract code into functions and call them inside the event listner
3) Dont use `innerHTML`